docs/layers/hexagon-layer.md

```diff
@@ -20,14 +20,14 @@ Inherits from all [Base Layer](/docs/layers/base-layer.md) properties.
 Primitive hexagon vertices as an array of six [lon, lat] pairs,
 in either clockwise or counter clouckwise direction.
 
-##### `dotRadius` (Number, optional)
+##### `radiusScale` (Number, optional)
 
 - Default: `1`
 
 Hexagon radius multiplier, between 0 - 1. It defines the gap between each hexagon.
```

Comment:
Could the sentence about "gap" otentially be confusing as this also depends on the centroids, which can be arbitrary (although we would expect them to be generated by a hex grid)?